charts/lagoon-logging/templates/logs-dispatcher.source-lagoon.configmap.yaml

```diff
@@ -22,5 +22,10 @@ data:
       routing_key   ""
       queue         "lagoon-logs:logs-dispatcher"
       durable       true
+      {{- if .Values.lagoonLogs.broker.tls.enabled }}
+      tls true
+      tls_ca_certificates ["/lagoon-ca.crt"]
```

Comment:
Does this need to be wrapped in `{{- if .Values.lagoonLogs.broker.tlsCA.enabled }}` or will it fail gracefully?

Reply:
was still a draft for a reason :sweat_smile: 

fixed now